commands/command_clean.go

```diff
@@ -22,7 +23,7 @@ import (
 //
 // If the object read from "from" is _already_ a clean pointer, then it will be
 // written out verbatim to "to", without trying to make it a pointer again.
-func clean(to io.Writer, from io.Reader, fileName string, fileSize int64) error {
+func clean(to io.Writer, from io.Reader, fileName string, fileSize int64) (nr, nw int64, err error) {
```

Comment:
The `nr` and `nw` values aren't very clear to me. It looks like the sizes are the blob size and the pointer size. What about something like `bSize` and `pSize`? Same goes for any `clean()` callers.